Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -148,6 +181,7 @@ class VDPortGroup : BaseDSC {
         }
         finally {
             $this.DisconnectVIServer()
+            Write-VerboseLog -Message $this.GetMethodEndMessage -Arguments @($this.DscResourceName)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.